### PR TITLE
complete fastify plugin

### PR DESCRIPTION
**Pull request by pavan-godspeed**
this PR has the fastify plugin code and need to review and merge.

### CODE REVIEW

**Review by mastersilv3r — COMMENTED** [review]
on first look. please dont include jode modules package lock dist folders included in your commit

**Review by mastersilv3r — CHANGES_REQUESTED** [review]
dont hardcode url.. use it from config.. like in express.. localhost etc should not go in code

File: plugins/fastify-as-http/README.md
Comment (siddarthvader) [inline plugins/fastify-as-http/README.md:1]: also update the root ReadME file to include fastify plugin @pavan-godspeed

File: plugins/fastify-as-http/package-lock.json
Comment (siddarthvader) [inline plugins/fastify-as-http/package-lock.json:1]: dont commit package-lock.json @pavan-godspeed

File: plugins/fastify-as-http/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                "dev": "tsc --watch",
+                "build": "tsc",
+                "prepublishOnly": "npm run build"
+        },
+        "license": "ISC",
+        "devDependencies": {
+                "@types/lodash": "^4.14.202",
+                "@types/qs": "^6.9.10",
+		"@types/fastify-jwt": "^0.8.1",
+                "lodash": "^4.17.21",
```
Comment (siddarthvader) [inline plugins/fastify-as-http/package.json:20]: i did not find use of lodash anywhere in your changes @pavan-godspeed

**Review by siddarthvader — CHANGES_REQUESTED** [review]
@pavan-godspeed i have added comments from first glance, i will be running this and test if this works meanwhile resolve these comments.

File: plugins/fastify-as-http/package-lock.json
Comment (pavan-godspeed) [reply plugins/fastify-as-http/package-lock.json:1]: untracked package-lock.json from git

File: plugins/fastify-as-http/package.json (hunk 1 above)
Comment (pavan-godspeed) [reply plugins/fastify-as-http/package.json]: removed unwanted (lodash) package from package.json.

**Pull request merged**